src/main/java/cart/controller/CartApiController.java

```diff
@@ -0,0 +1,33 @@
+package cart.controller;
+
+import cart.domain.Product;
+import cart.service.ProductService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api")
+@RequiredArgsConstructor
+public class CartApiController {
```

Comment:
```suggestion
public class ProductApiController {
```
가 아닌 `CartApiController` 인 이유가 있을까요? 🤔 

1단계에서 구현되어야할 내용은 `상품 관리 CRUD API 작성`이라서요

Reply:
CartApiController 가 적절하겠군요 
그렇게 변경할게요👍